apps/core/scenes/results/ResultsListItem.js

```diff
@@ -33,7 +33,9 @@ class ResultListItem extends React.Component<Props> {
   getSanitizedRouteItem = (routeItem: RouteItem) => {
     const timeSimpleFormat = 'H:mm';
 
-    const localArrival = this.parseDate(routeItem?.arrival?.localTime);
+    const localArrival =
+      routeItem?.arrival?.localTime &&
+      this.parseDate(routeItem?.arrival?.localTime);
```

Comment:
Why this change?

Reply:
It wasn't necessary, so I it's deleted now